postgres/server/init.sls

```diff
@@ -81,7 +81,7 @@ postgresql-cluster-prepared:
  {%- endif %}
     - cwd: /
     - env: {{ postgres.prepare_cluster.env }}
-    - runas: {{ postgres.user }}
+    - runas: {{ postgres.prepare_cluster.user }}
```

Comment:
Yes, the previous value was `postgres.prepare_cluster.user` so we should revert.